code/game/objects/effects/anomalies.dm

```diff
@@ -1,213 +1,350 @@
 //Anomalies, used for events. Note that these DO NOT work by themselves; their procs are called by the event datum.
 
+/// Chance of taking a step per second
+#define ANOMALY_MOVECHANCE 70
```

Comment:
Why have a define here? It's not referenced elsewhere and the `movechance` variable is descriptive enough to tell what the value is